packages/user-ui/src/pages/layout.rs

```diff
@@ -29,7 +29,7 @@ pub fn MainRootLayout(lang: Language) -> Element {
                                 CubeLoader {}
                             }
                         },
-                        div { class: "flex flex-col w-full min-h-lvh", Outlet::<Route> {} }
+                        div { class: "flex flex-col w-full min-h-lvh pt-[var(--header-height)]", Outlet::<Route> {} }
```

Comment:
```suggestion
                        div { class: "flex flex-col w-full min-h-lvh pt-(--header-height)", Outlet::<Route> {} }
```
https://tailwindcss.com/docs/padding#using-a-custom-value